e2e/chatqna/README.md

```diff
@@ -0,0 +1,27 @@
+# OPEA ChatQnA workload on OCP
+
```

Comment:
According to the architecture diagram, OPEA is the reference workload for the AI foundation, so suggest to put it into folder  workloads/OPEA/chatqna. @chaitanya1731 @hershpa  any comments?

Reply:
That sounds reasonable. A new workloads directory for reference workloads, some of which will be from OPEA.

Reply:
Yes. that sounds like a good idea.